tracker/dmlc_tracker/ssh.py

```diff
@@ -51,8 +96,13 @@ def submit(args):
                 h = h[:i]
             # hosts now contain the pair ip, port
             hosts.append((h, p))
+    return hosts
+
+def submit(args):
+    hosts = prepare_hosts(args.host_file)
+    envs = prepare_envs(args)
 
-    def ssh_submit(nworker, nserver, pass_envs):
+    def ssh_submit(nworker, nserver, dmlc_envs, addnl_envs):
```

Comment:
This feature can also be added to other submission scripts from local and ssh correct ?

Reply:
I see that currently only ssh mode passes additional env variables (like OMP_NUM_THREADS, KMP_AFFINITY..). 
MPI, Yarn and SGE are only passing dmlc environment variables, which are required to start training. Which leads me to believe either setting such environment variables is not useful/important in those modes (OR) these modes aren't being used much. 
Besides I don't have access to such clusters to verify that it works. I can clarify this in mxnet/tools/launch.py .